pkg/crc/preflight/preflight_checks_darwin.go

```diff
@@ -6,27 +6,39 @@ import (
 	"golang.org/x/sys/unix"
 	"io"
 	"net/http"
+	neturl "net/url"
 	"os"
 	"os/exec"
 	"os/user"
+	"path"
 	"path/filepath"
+	"strings"
+	"syscall"
 
+	"github.com/code-ready/crc/pkg/crc/constants"
 	"github.com/code-ready/crc/pkg/crc/logging"
 	"github.com/code-ready/crc/pkg/crc/oc"
+	dl "github.com/code-ready/crc/pkg/download"
 	crcos "github.com/code-ready/crc/pkg/os"
 )
 
 const (
 	virtualBoxDownloadURL   = "https://download.virtualbox.org/virtualbox/6.0.4/VirtualBox-6.0.4-128413-OSX.dmg"
 	virtualBoxMountLocation = "/Volumes/VirtualBox"
 
+	hyperkitDownloadURL       = "https://657-55985023-gh.circle-artifacts.com/0/hyperkit"
```

Comment:
@cfergeau I am just worry about how long this artifact going to stay on circle ci. According to the policy https://circleci.com/docs/2.0/artifacts/#artifacts-overview they also suggest not to rely on it for distribution side. Can we have it as part of hyperkit binary release or some other place which we own?

Reply:
I don't know what would be a good location to get the binary from? We probably should update it to something newer anyway. At this point, I'm thinking maybe it would make sense to include it in the crc_hyperkit_xxx.tar.xz tarball?
Do you have a suggestion for a place where we could upload the file?

Reply:
For now, the hyperkit binary is put next to the crc-machine-driver binary on github. Can be changed later if we have a better place.